ui/components/app/confirm-page-container/confirm-page-container.component.js

```diff
@@ -240,8 +247,14 @@ export default class ConfirmPageContainer extends Component {
             <PageContainerFooter
               onCancel={onCancel}
               cancelText={this.context.t('reject')}
+              hideCancel={isFailedTransaction}
               onSubmit={onSubmit}
-              submitText={this.context.t('confirm')}
+              submitText={
+                isFailedTransaction
+                  ? this.context.t('close')
+                  : this.context.t('confirm')
+              }
+              submitButtonType={isFailedTransaction ? 'default' : 'confirm'}
               disabled={disabled}
             >
               {unapprovedTxCount > 1 && (
```

Comment:
Should we hide the "Reject X" button as well when showing a failed transaction? It would be a bit confusing if that button didn't reject what is currently shown.

Reply:
The "Reject X" button is not shown.

Reply:
It is shown of there is more than one unapproved transaction. It is possible for that to be the case while viewing a failed transaction.